### PR TITLE
Adder html for section 1

### PR DESCRIPTION
**Pull request by MariaBgsl**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,5 +38,4 @@
   <!-- Подвал сайта-->
   <footer>Подвал сайта</footer>
 </body>
-
```
Comment (Evbgsl) [inline index.html:23]: Fix the formatting. Use Shift+Alt+F

File: assets/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# ng-free-zen
```
Comment (Evbgsl) [inline assets/README.md:1]: Move the file to the root folder

**Review by Evbgsl — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -10,7 +11,25 @@
   <!-- Шапка сайта-->
   <header>Шапка сайта</header>
   <!-- Секция 1-->
-  <section>Секция 1</section>
+  <section class="section" id="warranty">
```
Comment (Palych18) [inline index.html:14]: You need to replace the id value with "download"

File: index.html [hunk 4]
```diff
@@ -10,7 +11,25 @@
...
+    <div class="wrapper">
+      <div class="text-block">
+        <h1 class="title">Peaceful shopping mindful monwy</h1>
+        <p class="copy">See how we can help with making your shopping exeperience and money management more ZEN</p>
```
Comment (Palych18) [inline index.html:18]: If the element is large, it is better to move it to several lines
```
<p class="copy">
    See how we can help with making your shopping exeperience and money management more ZEN
</p>
```

File: index.html [hunk 5]
```diff
@@ -10,7 +11,25 @@
...
+        <div class="links">
+          <a class="link" href="https://apple.com">
+            <img src="./assets/icons/apple.svg" alt="Apple icon">
```
Comment (Palych18) [inline index.html:21]: It is recommended to close the single tag
```
<img src="./assets/icons/apple.svg" alt="Apple icon" />
```

**Review by Palych18 — COMMENTED** [review]
Fix

File: index.html [hunk 6]
```diff
@@ -10,7 +11,26 @@
   <!-- Шапка сайта-->
   <header>Шапка сайта</header>
   <!-- Секция 1-->
-  <section>Секция 1</section>
+  <section class="section" id="download">
+    <div class="wrapper">
+      <div class="text-block">
+        <h1 class="title">Peaceful shopping mindful monwy</h1>
+        <p class="copy">See how we can help with making your 
+          shopping exeperience and money management more ZEN</p>
```
Comment (Palych18) [inline index.html:19]:
```
<p class="copy">
    See how we can help with making your shopping exeperience and money management more ZEN
</p>
```

**Review by Palych18 — COMMENTED** [review]
Fix

**Pull request merged**